test/TensorFlowRuntime/tracer.swift

```diff
@@ -60,6 +60,59 @@ TracerTests.testAllBackends("Basic") {
   expectNearlyEqualWithScalarTensor(1.0, result2)
 }
 
+TracerTests.testAllBackends("AllDtypeSupport") {
+  func tracee<S: TensorGroup>(state: S, data: S) -> (S, S) {
+    return (state, data)
+  }
+
+  func traceAndRun<T>(_ v: T) -> (Tensor<T>, Tensor<T>)
+  where T: TensorFlowScalar & Numeric {
+    let state = Tensor<T>(v)
+    let data = Tensor<T>(v + 4)
+    let tracedFn = _graph(with: state, in: tracee)
+    return tracedFn(state, data)
+  }
+
+  func traceAndCheckNearlyEqual<T>(_ v: T)
+  where T: TensorFlowScalar & FloatingPoint & ExpressibleByFloatLiteral {
+    let (newState, newData) = traceAndRun(v)
+
+    expectNearlyEqualWithScalarTensor(v, newState)
+    expectNearlyEqualWithScalarTensor(v + 4, newData)
+  }
+
+  func traceAndCheckEqual<T>(_ v: T)
+  where T: TensorFlowScalar & Numeric & Comparable {
```

Comment:
Indent by 2 from `func`. Same in other places.

Reply:
Done.